File: backend/src/main/java/com/project2/global/app/AppConfig.java
package com.project2.global.app;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.context.annotation.Configuration;

import com.fasterxml.jackson.databind.ObjectMapper;

import lombok.Getter;

@Configuration
public class AppConfig {
	@Getter
	public static ObjectMapper objectMapper;

	@Autowired
	public void setObjectMapper(ObjectMapper objectMapper) {
		AppConfig.objectMapper = objectMapper;
	}
}

File: backend/src/main/kotlin/com/project3/global/app/AppConfig.kt
package com.project3.global.app

import com.fasterxml.jackson.databind.ObjectMapper
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.context.annotation.Configuration

@Configuration
class AppConfig {

    companion object {
        @JvmStatic
        lateinit var objectMapper: ObjectMapper
            private set
    }

    @Autowired
    fun setObjectMapper(objectMapper: ObjectMapper) {
        AppConfig.objectMapper = objectMapper
    }
}
